dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/Power.mo

```diff
@@ -9,20 +9,29 @@ within Dynawo.NonElectrical.Blocks.Continuous;
 * file, you can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 *
-* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
 */
 
 block Power "Outputs a power of the input"
   extends Modelica.Blocks.Interfaces.SISO;
 
   parameter Real N "Exponent of the power function";
+  parameter Boolean NInteger = true "For u < 0 : if true, u ^ N is calculable, if false, u ^ N is not calculable, y = 0 instead";
```

Comment:
Having a default value = true, makes it possible to change it to false later on?

Reply:
In ExTkjEdf1.mo (dynawo-rte branch 753), NInteger is false, the simulations are successful.

A default value can be overriden if the block is used in a Modelica model, this is different from changing the value from a .par file.

Reply:
But if we change it in the .par, we are not sure if we get the right value

Reply:
The simulation can be successful but the model behaves as if this parameter was true... better avoid default values.

Reply:
The default value has been removed.